consoles/sshVirtsh.pm

```diff
@@ -525,18 +525,18 @@ __END"
         my $vmx = sprintf('/vmfs/volumes/%s/openQA/%s.vmx', $bmwqemu::vars{VMWARE_DATASTORE} // 'datastore1', $self->name);
 
         # set default boot delay
-        $self->run_cmd("echo bios.bootDelay = \"10000\" >> $vmx", domain => 'sshVMwareServer');
+        $self->run_cmd("echo bios.bootDelay = \\\"10000\\\" >> $vmx", domain => 'sshVMwareServer');
```

Comment:
I would actually try to make it more readable:
```suggestion
        $self->run_cmd(q{echo 'bios.bootDelay = "10000"' >> $vmx}, domain => 'sshVMwareServer');
```
perl has these special quotes, so that you can both use `'` and `"` in the string, and now it's clear that the double quotes should land in the file.

Reply:
Oh sorry, it should be:
```
        $self->run_cmd(qq{echo 'bios.bootDelay = "10000"' >> $vmx}, domain => 'sshVMwareServer');
```
thanks @Martchus